1-js/02-first-steps/03-strict-mode/article.md

```diff
@@ -1,89 +1,89 @@
 ```
 
-Only comments may appear above `"use strict"`.
+Doar comentariile pot apărea deasupra `"use strict"`.
 ````
 
-```warn header="There's no way to cancel `use strict`"
-There is no directive like `"no use strict"` that reverts the engine to old behavior.
+```warn header="Nu există nicio modalitate de a anula `use strict`"
+Nu există o comandă de tipul `"no use strict"` care să readucă motorul de căutare ("browserul") la vechiul comportament.
 
-Once we enter strict mode, there's no going back.
+Odată ce am activat modul strict, nu mai există cale de întoarcere.
 ```
 
-## Browser console
+## Consola browserului
 
-When you use a [developer console](info:devtools) to run code, please note that it doesn't `use strict` by default.
+Atunci când folosiți [developer console](info:devtools) pentru a rula codul, vă rugăm să rețineți că aceasta nu folosește `use strict` în mod implicit.
 
-Sometimes, when `use strict` makes a difference, you'll get incorrect results.
+Uneori, când `use strict` face o diferență, veți obține rezultate incorecte.
 
-So, how to actually `use strict` in the console?
+Așadar, cum se utilizează de fapt `use strict` în consolă?
 
-First, you can try to press `key:Shift+Enter` to input multiple lines, and put `use strict` on top, like this:
+În primul rând, puteți încerca să apăsați `key:Shift+Enter` pentru a introduce mai multe linii, și să puneți `use strict` deasupra, astfel:
 
 ```js
-'use strict'; <Shift+Enter for a newline>
-//  ...your code
-<Enter to run>
+'use strict'; <Shift+Enter pentru o linie nouă>
+//  ...codul tău
+<Enter pentru a executa>
 ```
 
-It works in most browsers, namely Firefox and Chrome.
+Funcționează în majoritatea browserelor, mai ales în Firefox și Chrome.
 
-If it doesn't, e.g. in an old browser, there's an ugly, but reliable way to ensure `use strict`. Put it inside this kind of wrapper:
+Dacă nu funcționează, de exemplu, într-un browser vechi, există o modalitate inestetică, dar fiabilă de a asigura `use strict`. Puneți-l în acest tip de înveliș:
 
 ```js
 (function() {
   'use strict';
 
-  // ...your code here...
+  // ...codul tău...
 })()
 ```
 
-## Should we "use strict"?
+## Ar trebui să folosim comanda "use strict"?
 
-The question may sound obvious, but it's not so.
+Întrebarea poate părea banală, dar nu este așa.
 
-One could recommend to start scripts with `"use strict"`... But you know what's cool?
+Cineva ar putea recomanda să începeți scripturile cu `"use strict"`... Dar știi ce este grozav?
 
-Modern JavaScript supports "classes" and "modules" - advanced language structures (we'll surely get to them), that enable `use strict` automatically. So we don't need to add the `"use strict"` directive, if we use them.
+JavaScriptul modern suportă "classes" ("clase") și "modules" ("module") - structuri avansate de limbaj (vom ajunge cu siguranță la ele), care permit `use strict` în mod automat. Astfel, nu este nevoie să adăugăm comanda `"use strict"`, dacă le folosim.
```

Comment:
```suggestion
JavaScriptul modern suportă "classes" și "modules" -  structuri avansate de limbaj (vom ajunge cu siguranță la ele), care activează `use strict` automat. Astfel nu este nevoie să adăugăm directiva `"use strict"`, dacă le folosim.
```

Reply:
Idem cu termenii traduși prima dată.
Corect ”care activează `use strict` automat”.
Idem comanda vs directiva.

Reply:
Eu tot timpul fac o traducere inversă ca să verific cât de solidă este traducerea. În cazul cuvântului `comandă`, tradus invers vine `command` în loc de `directive`.
Cel puțin asta a fost logica mea. 😁